Explorer/Assets/DCL/Character/CharacterCamera/IExposedCameraData.cs

```diff
@@ -19,6 +20,8 @@ public interface IExposedCameraData : IExposedTransform
 
         CanBeDirty<bool> PointerIsLocked { get; }
 
+        EntityProxy CameraEntityProxy { get; }
```

Comment:
It's good to generalise Proxy<T>, I see you use it in multiple places: EntityProxy, MainPlayerAvatarBaseProxy. And their contracts differs, so it would be good to have it as one

Reply:
Tackled the needed refactor unifying the "proxy" usages into a new `ObjectProxy<T>` at [this commit](https://github.com/decentraland/unity-explorer/pull/340/commits/e58d6c8346f2450f1f8bfa137c2e610704d62991)